script/lazytox.py

```diff
@@ -1,238 +0,0 @@
-#!/usr/bin/env python3
```

Comment:
I'd like to keep this script. We can rename it instead maybe?

Reply:
Yeah sure, suggestion for naming?

Reply:
`lint_and_test.py`